imap/ctl_mboxlist.c

```diff
@@ -614,6 +614,9 @@ static int dump_cb(const mbentry_t *mbentry, void *rockp)
     /* char *uniqueid; */
     json_object_set_new(jobj, "uniqueid", json_string(mbentry->uniqueid));
 
+    /* char *jmapid; */
+    json_object_set_new(jobj, "jmapid", json_string(mbentry->jmapid));
```

Comment:
nit: for consistency with the JMAP API specs, this should be camel-case: `jmapId`?

Reply:
Fixed